docs/how-to-enable-new-languages.md

```diff
@@ -281,3 +281,101 @@ Once these are in place, you should be able to run `npm run develop` to view you
 
 > [!ATTENTION]
 > While you may perform translations locally for the purpose of testing, we remind everyone that translations should _not_ be submitted through GitHub and should only be done through Crowdin. Be sure to reset your local codebase after you are done testing.
+
+# Deploying New Languages on `/news`
+
+## Add a YAML File for Trending Articles
+
+News sources trending links and article titles from the [CDN repo](https://github.com/freeCodeCamp/cdn) during the build and adds them to the footer.
+
+Clone the CDN repo and create a new branch.
+
+In the [`build/universal/trending`](https://github.com/freeCodeCamp/cdn/tree/main/build/universal/trending) directory, create a new file and name it `language.yaml`. For example, if you are launching Dothraki News, name the file `dothraki.yaml`.
+
+Then copy the contents of the [`english.yaml`](https://github.com/freeCodeCamp/cdn/blob/main/build/universal/trending/english.yaml) trending file and paste it into the new YAML file you just created.
+
+The contents will look something like this:
+
+```yaml
+article0title: 'Learn JavaScript'
+article0link: 'https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/'
+article1title: 'Linux ln Example'
+article1link: 'https://www.freecodecamp.org/news/linux-ln-how-to-create-a-symbolic-link-in-linux-example-bash-command'
+article2title: 'JS document.ready()'
+article2link: 'https://www.freecodecamp.org/news/javascript-document-ready-jquery-example/'
+article3title: ...
+article3link: ...
+  ...
+```
+
+Open a PR to the CDN repo with these changes and tag `@freeCodeCamp/dev-team` for review.
```

Comment:
```suggestion
Open a PR to the CDN repo with these changes for review.
```

We don't need to tag - the CODEOWNERS auto-requests reviews in a round robin style.